### ISSUE TITLE
Add `ctx.BindJson<'T>(jsonSerializerOptions)`

### ISSUE THREAD

**Issue by pkese** [Lanayx/Oxpecker#34]
It would be nice if [context.BindJson](https://github.com/Lanayx/Oxpecker/blob/573ca976dce526ca4e2ebd30d6592c4248a4c710/src/Oxpecker/HttpContextExtensions.fs#L357) would allow passing custom `JsonSerializerOptions`.

**Comment by Lanayx** [comment]
@pkese have you seen the json custom configuration docs https://lanayx.github.io/Oxpecker/src/Oxpecker/#writing-json ?

**Comment by pkese** [comment]
Yes, I have, but I have different parts of API are expecting different types of data:  
one part of my API is public, the other part is exposed to F# client sending Thoth.Json compliant data, so I need two sets of JsonSerializerOptions

**Comment by Lanayx** [comment]
I see. I think that current helpers like `BindJson` are just for the most common scenarios, but it should be very easy to add customize, for example

```fsharp
module My =
    let anotherSerializer = SystemTextJsonSerializer(myOptions)
```
and then
```fsharp
[<Extension>]
static member WriteJson2<'T>(ctx: HttpContext, value: 'T) =
    My.anotherSerializer.Serialize(value, ctx, false)
```
so give this is easy achievable I'm reluctant to extend API
